src/frida-tvos.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __FRIDA_TVOS_H__
+#define __FRIDA_TVOS_H__
+
+#ifdef HAVE_TVOS
+#include <Availability.h>
+#undef __TVOS_PROHIBITED
+#define __TVOS_PROHIBITED
```

Comment:
Another approach would be to to generate a new `sysroot` with these markers removed.

https://github.com/palera1n/palera1n/blob/d17af1fbc7efc7d525f1be7fd448fd158cb65251/.github/workflows/build.yml#L171-L173